edx_sga/sga.py

```diff
@@ -449,27 +469,43 @@ def prepare_download_submissions(self, request, suffix=''):  # pylint: disable=u
         user = self.get_real_user()
         require(user)
         zip_file_ready = False
+        location = unicode(self.location)
 
         if self.is_zip_file_available(user):
+            log.info("Zip file already available for block: %s for staff: %s", location, user.username)
             assignments = self.get_sorted_submissions()
             if assignments:
                 last_assignment_date = assignments[0]['timestamp'].astimezone(pytz.utc)
+                log.info(
+                    "Last assignment upload time: %s for block: %s for staff: %s",
+                    last_assignment_date,
+                    location,
+                    user.username
+                )
```

Comment:
I think this log message can be removed. The message at line 492 gives the user the same information